cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -26,6 +26,9 @@
     add_container_page_publishing_info
 )
 from contentstore.tests.utils import CourseTestCase
+from lms_xblock.mixin import (
```

Comment:
Please run isort on this file to move the imports to the correct places. I'll forward the isort email message, as it likely was sent before you arrived.